static/style.css

```diff
@@ -42,7 +42,7 @@ nav ol {
   padding-left: 1rem;
 }
 
-main {
+#main-output {
```

Comment:
another difference betweem the static page created by `make docs` and the `dynamic.html` used by `dex web`
is that static uses as `div` and the dynamic uses a `main`.
But both have the id `#main-output`.
So this just makes there styling consistent.

Reply:
I mean, we can also just change the tag to always be `main`. But this is good enough

Reply:
Yeah, I was going to, but I couldn't fund where the tag was defined in `RenderHtml.hs`